checkstyle.xml

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0"?>
```

Comment:
No need for checkstyle enforcing

Reply:
If this is meant to be merged into Fabric API, then it needs checkstyle enforcing. If it's not meant to be merged into Fabric API, then it shouldn't be here.